### ISSUE TITLE
[Bug] Connect to CnosDB through dbeaver, tables information cannot be viewed through ui

### ISSUE THREAD

**Issue by yukkit** [cnosdb/cnosdb#1518]
### Describe the bug

![image](https://github.com/cnosdb/cnosdb/assets/28214641/a321238b-070d-48df-bc3b-d96ec1cc67e0)

```
Cannot invoke "cfjd.org.apache.arrow.vector.FieldVector.getField()" because "originalVector" is null
```

### To Reproduce

1. 创建驱动

类名： org.apache.arrow.driver.jdbc.ArrowFlightJdbcDriver
URL模版：jdbc:arrow-flight-sql://{host}:{port}?tenant={server}&useEncryption=false

![image](https://github.com/cnosdb/cnosdb/assets/28214641/78916055-eb39-4435-95ec-62d316a22cbb)

2. 创建数据库连接

![image](https://github.com/cnosdb/cnosdb/assets/28214641/a5737e2f-6945-419f-bd49-fc8def2d2ccd)

_No response_

### Additional context

driver: flight-sql-jdbc-driver-12.0.0.jar

url: jdbc:arrow-flight-sql://localhost:8904?user=root&tenant=cnosdb&useEncryption=false

**Comment by yukkit** [comment]
This problem is caused by the arrow flight sql jdbc library,  https://github.com/apache/arrow/issues/36880 , which requires the server's `getTables`/`getSchemas` and other api to return the hard-coded name in the jdbc library, for example: TABLE_SCHEM must be db_schema_name.

To solve this problem, we need to make a compromise.

**Comment by Benxiaohai001** [comment]
The issue has been fixed,version: cnosdb 2.3.2, revision 28eb92612149ac14337236f81c1e4eeb1e0e013b
